react/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "redux-loop-react",
```

Comment:
call this redux-loop/react. then you can import it like that

Reply:
that's illegal in package.json, it can only have `/` if it's scoped